### PR TITLE
Code Challenge #3

### PR DESCRIPTION
**Pull request by johngwells**

### CODE REVIEW

File: isUnique/isUnique.js [hunk 1]
```diff
@@ -3,9 +3,30 @@
 * Extra Credit - Answer this question - What if you cannot use additional data structures?
 */
 
+// Create a loop to check individual characters and check to see if it was used before
+// charAt() method returns the character at a specific index.
+// str.indexOf('a', 5) find the first occurance of a letter in a string. ex: a, start position at 5
 const isUnique = (str) => {
```
Comment (ryan-hamblin) [inline isUnique/isUnique.js:9]: Using nested loops can get heavy, but this is a very common solution to this problem. ( no need to respond back to me with my next question ) Could you think of a way to do this without nesting forLoops?

File: longestString/longestString.js [hunk 2]
```diff
@@ -1,4 +1,19 @@
...
+
+ // Count the letters in each array element
+ // Count the number of words in an array
+ // compare the letterCount to Words in Array
+let array = ['hi', 'hello', 'welcome'];
+let length = 0;
+let longest;
+
+for (let i = 0; i < array.length; i++) {
+	if (array[i].length > length) {
```
Comment (ryan-hamblin) [inline longestString/longestString.js:14]: Watch your code indentation here. Formatters and linters will bark at this :)

File: reverseCase/reverseCase.js [hunk 3]
```diff
@@ -3,5 +3,20 @@
  * Example: 'Hello World' -> 'hELLO wORLD'
  * Assume that each string will contain only spaces and letters.
  */
+const newString = [];
 
+const lowerUpper = (string) => {
+  for (let i = 0; i < string.length; i++) {
+    const character = string[i];
+    if (character === character.toLowerCase()) {
```
Comment (ryan-hamblin) [inline reverseCase/reverseCase.js]: Awesome. I love the readability of your code. It is nice and clean, and it flows really well. I would be glad to receive this file if I was working on a team with you! Keep it up.

**Review by ryan-hamblin — COMMENTED** [review]
All three algorithms look great! Keep up the good work. Reflect on the comments that I've made and see if you can't brainstorm a solution to isUnique without having to nest for loops :)

File: reverseCase/reverseCase.js (hunk 3 above)
Comment (johngwells) [reply reverseCase/reverseCase.js]: Thanks appreciate the feedback! Glad to be working & learning from you.

**Pull request closed without merging**